sdk/core/azure-mgmt-core/azure/mgmt/core/tools.py

```diff
@@ -34,7 +34,9 @@
     "(/providers/(?P<namespace>[^/]+)/(?P<type>[^/]*)/(?P<name>[^/]+)(?P<children>.*))?"
 )
 
-_CHILDREN_RE = re.compile("(?i)(/providers/(?P<child_namespace>[^/]+))?/" "(?P<child_type>[^/]*)/(?P<child_name>[^/]+)")
+_CHILDREN_RE = re.compile(
+    "(?i)(/providers/(?P<child_namespace>[^/]+))?/" "(?P<child_type>[^/]*)/(?P<child_name>[^/]+)"
+)  # pylint: disable=implicit-str-concat
```

Comment:
Should this be ignored? Why it the line structured like this?
I'm also curious why pylint didn't have issue with the line above on 32-34

Reply:
I was ignoring it to avoid changing the code, but I can fix it! I didn't notice the previous line, but I checked it, it didn't trigger pylint bc the strings are on 2 separate lines instead of one